.github/workflows/avm.res.desktop-virtualization.scaling-plan.yml

```diff
@@ -0,0 +1,83 @@
+name: "avm.res.desktop-virtualization.scaling-plan"
```

Comment:
Hey @moisesjgomez, thank you for your contribution. Please note that for a new module, also the corresponding line in the `.github/CODEOWNERS` file needs to be uncommented, and the corresponding teams created.
@eriqua would you happen to have the links in the specs handy that describe which teams are needed and how they need to be added where? I tried to find it, but failed.

Reply:
Hi @moisesjgomez for guidelines on how to configure teams please refer to  https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships